README.md

```diff
@@ -76,9 +76,9 @@ This will not uninstall any apps that you installed through Pi-Apps.
 ```
 </details>
 
-## To run Pi-Apps
```

Comment:
"To" is capitalized because it starts a complete phrase, while Pi-Apps is capitalized because it is a proper name.
I suspect that your reason to capitalize "Run" is to capitalize every word, like the "To Update Pi-Apps" title below it.
Honestly, in the name of consistency, I would prefer having "update" lowercase instead of this change.
Any comments?

Reply:
agreed with Botspot, update should be lowercase